scripts/create-symlinks.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Usage in the project root folder:
+# npm run init -- PACKAGE_NAME
+
+
+if [ -z "$1" ]
+	then
+		echo "Please provide a package name!";
+		exit 1;
+fi
+
+packageDir="./packages/$1";
+# Just in case package folder doesn't exist yet.
```

Comment:
Should it error if the package dir does exist?

Reply:
We may wanna create symlinks after creating the folder. So let's keep it just in case.